events/publish.go

```diff
@@ -80,7 +86,9 @@ loop:
 func processEvents(bus pubsub.Bus, events []abci.Event) {
 	for _, ev := range events {
 		if mev, ok := processEvent(ev); ok {
-			bus.Publish(mev)
+			if err := bus.Publish(mev); err != nil {
+				panic(err)
```

Comment:
should we handle err to gracefully shutdown?

Reply:
```go
bus.Close()
return
```